checklists/alz_checklist.en.json

```diff
@@ -336,13 +336,13 @@
         {
             "category": "Identity and Access Management",
             "subcategory": "Landing zones",
-            "text": "Configure Identity network segmentation through the use of a virtual Network and peer back to the hub. Providing authentication inside application landing zone (legacy).",
+            "text": "The identity subscription should have a virtual network that is peered to the hub virtual network in the platform subscription.",
             "waf": "Security",
             "guid": "9cf5418b-1520-4b7b-add7-88eb28f833e8",
             "id": "B04.01",
             "severity": "Medium",
-            "training": "https://learn.microsoft.com/azure/architecture/example-scenario/identity/adds-extend-domain",
-            "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/identity-access-landing-zones#managed-identities"
+            "training": " https://learn.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/identity-access-landing-zones",
```

Comment:
👍

Reply:
Do we need `training` URL as `link` has the same URL?

Reply:
I think one of the "style guide" things we've talked about is that Training should always be a training module, not just a page on the topic?  But I think there are some places where there is not specific training.

But I agree we shouldn't double up on the same link.